app/subscriber/src/features/my-reports/edit/ReportEditForm.tsx

```diff
@@ -198,16 +229,65 @@ export const ReportEditForm = React.forwardRef<HTMLDivElement | null, IReportEdi
             }
           }}
         />
+        <Modal
+          headerText="Unlock Report"
```

Comment:
We can now unlock a report.  I don't think this is wise, but it may resolve some odd challenges we're having testing.